django_migration_linter/management/commands/lintmigrations.py

```diff
@@ -128,70 +136,61 @@ def handle(self, *args, **options):
 
-        ignore_name_contains = options["ignore_name_contains"] or config_parser.get(
-            CONFIG_NAME, "ignore_name_contains", fallback=None
-        )
-        ignore_name = options["ignore_name"] or config_parser.get(
-            CONFIG_NAME, "ignore_name", fallback=None
-        )
-        include_name_contains = options["include_name_contains"] or config_parser.get(
-            CONFIG_NAME, "include_name_contains", fallback=None
-        )
-        include_name = options["include_name"] or config_parser.get(
-            CONFIG_NAME, "include_name", fallback=None
-        )
-        include_apps = options["include_apps"] or config_parser.get(
-            CONFIG_NAME, "include_apps", fallback=None
-        )
-        exclude_apps = options["exclude_apps"] or config_parser.get(
-            CONFIG_NAME, "exclude_apps", fallback=None
-        )
-        database = options["database"] or config_parser.get(
-            CONFIG_NAME, "database", fallback=None
-        )
-        cache_path = options["cache_path"] or config_parser.get(
-            CONFIG_NAME, "cache_path", fallback=None
-        )
-        no_cache = options["no_cache"] or config_parser.getboolean(
-            CONFIG_NAME, "no_cache", fallback=None
-        )
-        applied_migrations = options["applied_migrations"] or config_parser.getboolean(
-            CONFIG_NAME, "applied_migrations", fallback=None
-        )
-        unapplied_migrations = options[
-            "unapplied_migrations"
-        ] or config_parser.getboolean(
-            CONFIG_NAME, "unapplied_migrations", fallback=None
-        )
-        exclude_migration_tests = options[
-            "exclude_migration_tests"
-        ] or config_parser.get(CONFIG_NAME, "exclude_migration_tests", fallback=None)
-        quiet = options["quiet"] or config_parser.get(
-            CONFIG_NAME, "quiet", fallback=None
-        )
-        warnings_as_errors = options["warnings_as_errors"] or config_parser.getboolean(
-            CONFIG_NAME, "warnings_as_errors", fallback=None
-        )
+        config = {key: options[key] for key in keys}
+
+        if toml and os.path.exists(PYPROJECT_TOML):
+            pyproject_toml = toml.load(PYPROJECT_TOML)
+            section = pyproject_toml.get("tool", {}).get(CONFIG_NAME, {})
+            config.update({key: section[key] for key in keys if key in section})
+        else:
+            config_parser = configparser.ConfigParser()
+            try:
+                config_parser.read(DEFAULT_CONFIG_FILES, encoding="utf-8")
+            except configparser.ParsingError as err:
+                if err.source == PYPROJECT_TOML:
+                    logger.error("Install toml for full pyproject.toml support")
+                raise
+            config.update(
+                {
+                    key: config_parser.get(CONFIG_NAME, key, fallback=None)
+                    for key in keys
+                    if config_parser.get(CONFIG_NAME, key, fallback=None)
```

Comment:
`config_parser.get(CONFIG_NAME, key, fallback=None)` is not very readable in the first place,
Let's avoid putting it twice in the same dict comprehension and abuse of this construct :)